carma/rviz/carma_default.rviz

```diff
@@ -639,41 +608,46 @@ Visualization Manager:
   Value: true
   Views:
     Current:
-      Angle: 0.639999986
-      Class: rviz/TopDownOrtho
```

Comment:
Can we keep the default view as TopDownOrtho. Having to change it will slow down launch times a bit.